dask/delayed.py

```diff
@@ -141,6 +141,7 @@ def to_task_dask(expr):
 
     Examples
     --------
+    # TODO: Fix this doctest
```

Comment:
Don't quite know how to fix this one.
```python
142     Examples
143     --------
        # TODO: Fix this doctest
144     >>> import dask
145     >>> a = delayed(1, 'a')
146     >>> b = delayed(2, 'b')
147
148     >>> task, dask = to_task_dask([a, b, 3])    
UNEXPECTED EXCEPTION: UserWarning('The dask.delayed.to_dask_dask function has been Deprecated in favor of unpack_collections')
Traceback (most recent call last):
  File "/Users/zhengnan/miniforge3/envs/dask-dev/lib/python3.8/doctest.py", line 1336, in __run
    exec(compile(example.source, filename, "single",
  File "<doctest dask.delayed.to_task_dask[3]>", line 1, in <module>
  File "/Users/zhengnan/Desktop/open-source/dask/dask/delayed.py", line 160, in to_task_dask
    warnings.warn(
UserWarning: The dask.delayed.to_dask_dask function has been Deprecated in favor of unpack_collections
```

Reply:
This one can probably stay skipped since the method is deprecated.